test/dotnet/Integration/RedisListTriggerTestFunctions.cs

```diff
@@ -41,6 +42,30 @@ public static class RedisListTriggerTestFunctions
             logger.LogInformation(IntegrationTestHelpers.GetLogValue(entry));
         }
 
+        [FunctionName(nameof(ListTrigger_Batch_RedisValue))]
+        public static void ListTrigger_Batch_RedisValue(
+            [RedisListTrigger(localhostSetting, nameof(ListTrigger_Batch_RedisValue), pollingIntervalInMs: pollingIntervalShort, count: count)] RedisValue[] entry,
```

Comment:
As I think we discussed a few weeks ago, the property name "count"is relatively vague and not really consistent with our other triggers. This controls the size of the batch right? In that case, the name of the setting for Event Hubs is maxBatchSize and that seems appropriate here too.

Reply:
This has been merged and this PR has been updated